### PR TITLE
Emoji in comparison table

### PR DESCRIPTION
**Pull request by kalabiyau**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless this is a trivial change, please include a CHANGELOG entry.<br />
Run <code>osc vc</code> in the <code>package</code> directory to add one.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ikapelyukhin** [comment]
Green check mark maybe instead of the white one?

**Comment by ikapelyukhin** [comment]
I.e.: :heavy_check_mark:

File: README.md [hunk 1]
```diff
@@ -153,7 +153,8 @@ RMT is replacing some functionality of [SMT](https://github.com/SUSE/smt). Follo
 |Transfer registration data to SCC|:white_check_mark:|:x:|
 |Reporting|:white_check_mark:|:x:|
 |Custom TLS certificates for web-server|:white_check_mark:|:white_check_mark:|
-|Web-server|Apache2|Nginx|
+|Webserver|Apache2|Nginx|
+|Database|mariadb|mariadb|
```
Comment (ikapelyukhin) [inline README.md]: `MariaDB`

File: README.md [hunk 2]
```diff
@@ -129,31 +128,32 @@ RMT is replacing some functionality of [SMT](https://github.com/SUSE/smt). Follo
...
-|YaST management wizard|yes|no|
-|Client management|yes|no|
-|RedHat support (Extended Support)|yes|no<sup>[2](#res)</sup>|
-|Files deduplication|yes|yes|
-|Data transfer from SMT to RMT|-|yes|
-|Transfer registration data to SCC|yes|no|
-|Reporting|yes|no|
-|Custom TLS certificates for web-server|yes|yes|
-|Web-server|Apache2|Nginx|
+|Available on SLES11|:white_check_mark:|:x:|
```
Comment (kalabiyau) [inline README.md]: note to self: check if available for Leap 42 and friends

Comment (digitaltom) [reply README.md]: rmt-server itself is available on 12, but not the yast module. So it involves some manual setup to get it running.

Comment (ikapelyukhin) [reply README.md]: There's no documentation on the manual setup (on how to generate the SSL certs and such), though.

Comment (kalabiyau) [reply README.md]: Customer only can get RMT on SLES12 via non-supported channels, so it is effectively not available from my point of view. You can install it, for sure, but with manual configuration and SSL setup.

**Comment by kalabiyau** [comment]
@ikapelyukhin check now - any better?

**Comment by lagartoflojo** [comment]
I made a minimal alignment change, just remove my commit if you no like :)

**Pull request merged**